libs/net/dal/Services/ReportService.cs

```diff
@@ -123,6 +123,7 @@ public IEnumerable<Report> FindMyReports(int userId)
             .Include(f => f.Owner)
             .Include(r => r.Template).ThenInclude(t => t!.ChartTemplates)
             .Include(r => r.Sections)
+            .Include(r => r.Instances)
```

Comment:
Please don't do this.  Think about how much data will be returned each time.

Reply:
Many reports run every day.  This means in a year this would return 365 extra items.  Of which none will ever be viewed again.